internal/testgen/gen_compares.go

```diff
@@ -22,41 +22,52 @@ func (g ComparesTestsGenerator) TemplateData() any {
 	)
 
 	boolOps := []token.Token{token.LAND, token.LOR}
-	ignored := make([]Assertion, 0, len(boolOps)*2)
+	ignoredAssertions := make([]Assertion, 0, len(boolOps)*2)
 	for _, tok := range boolOps {
-		ignored = append(ignored,
+		ignoredAssertions = append(ignoredAssertions,
 			Assertion{Fn: "True", Argsf: fmt.Sprintf("c %s d", tok)},
 			Assertion{Fn: "False", Argsf: fmt.Sprintf("d %s c", tok)},
 		)
 	}
 
+	var invalidAssertions []Assertion //nolint:prealloc
+	for checker, oppositeChecker := range map[string]string{
+		"True": "False",
```

Comment:
loop and map doesn't make sense in such case
this is only increase complexity of support

please revert it or do in the same way as `append` below

Reply:
This one was there, because #111 had another commit to add things 1eb5934a8f43d09422d826ac93987ae2e40e882d

I will revert sure